pkgs/top-level/python-packages.nix

```diff
@@ -15341,6 +15402,25 @@ in {
     };
   };
 
+  graphviz = buildPythonPackage rec {
+    name = "graphviz-${version}";
+    version = "0.5.2";
+    disable = isPy3k;
```

Comment:
According to the [project page](https://github.com/xflr6/graphviz): “This package runs under Python 2.6, 2.7, and 3.3+”.

The [setup.py](https://github.com/xflr6/graphviz/blob/289f0ea41c9d1e1ff1b51225ba5a8820dc019c89/setup.py) has:

```python
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.6',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.3',
        'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: 3.5',
```

as classifiers.

Any reason to disable this derivation for python3?